_episodes/10-prog.md

```diff
@@ -180,6 +180,85 @@ and translates any special characters in the values
 into their escaped equivalents
 so that they are safe to use.
 
+We can also use `sqlite3`'s cursor to make changes to our database,
+such as inserting a new name.
+For instance, we can define a new function called `add_name` like so:
+
+~~~
+def add_name(database_file, new_person):
+    query = "INSERT INTO Person VALUES" + repr(person_details) + ";"
```

Comment:
Can we avoid those constructs? They lead to SQL injections

Reply:
Excellent point, fixed in the latest commit. Thanks!